Create ui-c8y-1021-22-0-allows-to-add-a-tab-to-the-alarm-detail-view.md (#2711)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-0-allows-to-add-a-tab-to-the-alarm-detail-view.md

---
date: ""

> title: allows to add a tab to the alarm detail view

Author: The Claude AI suggested to use the following title:
```suggestion
title: Alarm details can now be extended with custom tabs
```

> title: Alarm details can now be extended with custom tabs

Reviewer: ```suggestion
title: Added a hook service to allow extension of alarm details view with custom tabs
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59811
version: 1021.22.0
---

> allows to add a tab to the alarm detail view

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs to display alarm related information. This allows integrating 3rd party data or visualizations that provide additional context for an alarm. Existing alarm details will not be affected, the new custom tabs will be added in addition to the already existing tabs.
```

> In some cases, it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs to display alarm related information. This allows integrating 3rd party data or visualizations that provide additional context for an alarm. Existing alarm details will not be affected, the new custom tabs will be added in addition to the already existing tabs.

Reviewer 2: ```suggestion
In some cases it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs to display alarm-related information. This allows integrating third-party data or visualizations that provide additional context for an alarm. Existing alarm details will not be affected, the new custom tabs will be added in addition to the already existing tabs.
```

> In some cases it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs to display alarm-related information. This allows integrating third-party data or visualizations that provide additional context for an alarm. Existing alarm details will not be affected, the new custom tabs will be added in addition to the already existing tabs.

Reviewer: ```suggestion
To provide more flexibility for customizing alarm detail views, hook services have been added to extend alarm details with custom tabs, along with the inline tabs component. This enables developers to display additional context-specific information for alarms or easily integrate third-party data and visualizations in the view. The existing alarm details functionality remains unchanged, while custom tabs appear as additional options in the details view.
```

Reviewer: @janhommes  pls check the proposed changes. I have tried keeping it similar to previous one https://cumulocity.com/docs/change-logs/#ui-c8y-1020-14-0-mtm-60181-add-hook-service-for-alarm-detail-button-and-alarm
